llvm/lib/Target/RISCV/RISCVSchedSiFive7.td

```diff
@@ -1041,6 +1044,7 @@ def : SiFive7AnyToGPRBypass<ReadCTZ32>;
 def : ReadAdvance<ReadCPOP, 0>;
 def : ReadAdvance<ReadCPOP32, 0>;
 def : SiFive7AnyToGPRBypass<ReadORCB>;
+def : SiFive7AnyToGPRBypass<ReadIMinMax>;
```

Comment:
Need to add WriteIMinMax to the list in SiFive7AnyToGPRBypass

Reply:
Updated